### DISCUSSION TITLE
[FeatureRequest] Allow using variables in unit field

### DISCUSSION THREAD

**Discussion by szerwi** [grafana/grafana#38259] [Dashboards, Panels & Visualizations]
**What would you like to be added**:
Ability to use variables in panel's "Unit" field.

**Why is this needed**:
Currently it is possible to put a variable in "Display name" field, but the same is not possible in "Unit" field
![image](https://user-images.githubusercontent.com/19311511/114415517-0c6dbd80-9bb0-11eb-8cb3-d4922a6be879.png)

**Comment by pkolyvas** [comment]
We're in the process of moving Grafana feature requests to [GitHub Discussions](https://community.grafana.com/t/were-moving-grafana-feature-requests-to-github-discussions/50341/3). This issue is being moved to [Discussions](https://github.com/grafana/grafana/discussions) or has been moved from our issues list, depending on where you're seeing this message. 

 We hope to use the discussion thread to continue building use cases, and additional evidence based on engagement/voting, in order to help us prioritize this feature request later on down the road. 

As with all things Grafana, we value your input and participation in this project and appreciate your patience during this transitional period. Looking forward to the discussion!

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). Due to the age and number of responses to this discussion, we are deciding to close it. If this is something you would like to see in Grafana, feel free to open an [issue](https://github.com/grafana/grafana/issues) so the discussion can continue. Thank you!
